homeassistant/components/sensor/usps.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
@J-CMartin If the cookie is old, the intended behavior of `myusps` is to attempt to fetch the page with the cookie, if it fails, login again (get a new cookie) and then retry to fetch the page with the new cookie. If there's a bug in that process, we should solve it in `myusps`.